tools/scripts/build/download-trending.ts

```diff
@@ -44,4 +44,7 @@ const download = async () => {
   }
 };
 
-void download();
+void download(clientLocale);
+// TODO: remove the need to fallback to english once we're confident it's
+// unnecessary (client/i18n/config.js will need all references to 'en' removing)
+void download('english');
```

Comment:
Won't make much of a difference in the end, but maybe slightly more efficient for English builds:

```suggestion
if (clientLocale !== 'english') void download('english');
```